app/views/schemes/_scheme_list.html.erb

```diff
@@ -26,7 +26,12 @@
           <% row.with_cell(text: scheme.owning_organisation&.name) %>
           <% row.with_cell(text: scheme.id_to_display) %>
           <% row.with_cell(text: scheme.locations.visible&.count) %>
-          <% row.with_cell(text: status_tag_from_resource(scheme)) %>
+          <% row.with_cell do %>
+            <%= status_tag_from_resource(scheme) %>
+            <% unless scheme.has_active_locations? %>
+              <%= content_tag(:div, "No currently active locations", class: "app-!-colour-muted", style: "margin-top: 10px") %>
+            <% end %>
```

Comment:
`has_active_locations?` also checks if scheme itself is active. I feel like the name for the method is a bit misleading.
Is is a conscious decision to only show this tag on active schemes?

Reply:
Yes its a conscious decision to only show the tag on active schemes, otherwise this tag would be shown on any scheme incomplete or deactivated which is not as designed in the ticket.

Reply:
I can rename it to `active_and_has_active_locations?`

Reply:
That could work